lerobot/common/policies/act/modeling_act.py

```diff
@@ -200,15 +208,17 @@ def __init__(self, config: ACTConfig):
         self.config = config
         # BERT style VAE encoder with input tokens [cls, robot_state, *action_sequence].
         # The cls token forms parameters of the latent's distribution (like this [*means, *log_variances]).
-        self.use_input_state = "observation.state" in config.input_shapes
+        self.expected_state_keys = [k for k in config.input_shapes if k.startswith("observation.state")]
```

Comment:
I think this is repeated right? It should only by in the `__init__`?

Reply:
Yes sorry, I just hacked things together to get a minimum working version, will clean things up.

Reply:
Actually, this is needed here for `self.use_input_state` and `self.state_shape`

Reply:
Hmm GH review are making it hard for me to see your changes with this comment. Will start another thread.